test/services/foreman_puppet/host_info_providers/config_groups_info_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_puppet_helper'
+
+module ForemanPuppet
+  class ConfigGroupInfoTest < ActiveSupport::TestCase
+    let(:cg_info) { HostInfoProviders::ConfigGroupsInfo.new(host) }
+
+    context 'with Puppet Host' do
+      let(:config_group) { FactoryBot.create(:config_group) }
+      let(:host) do
+        FactoryBot.build(:host, :with_puppet_enc,
+          location: taxonomies(:location1),
+          organization: taxonomies(:organization1),
+          operatingsystem: operatingsystems(:redhat),
+          config_groups: [config_group])
+      end
+
+      it 'adds config_groups to host parameters' do
+        assert_equal(cg_info.host_info['parameters']['foreman_config_groups'], [config_group.name])
```

Comment:
expected value should be passed as first argument, if I understand correctly, `[config_group.name]` is the expected value and `cg_info.host_info['parameters']['foreman_config_groups']` is the actual value, right?

```suggestion
        assert_equal([config_group.name], cg_info.host_info['parameters']['foreman_config_groups'])
```

Reply:
Ah, I checked that but apparently not good enough.